<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Графический редактор</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/Icon.ico</normaloff>:/images/Icon.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QWidget" name="gridLayoutWidget">
         <layout class="QGridLayout" name="gridLayout" columnminimumwidth="0,80">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="spacing">
           <number>2</number>
          </property>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/04.png</normaloff>:/images/04.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/01.png</normaloff>:/images/01.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/08.png</normaloff>:/images/08.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/02.png</normaloff>:/images/02.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButton_5">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/03.png</normaloff>:/images/03.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_6">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/06.png</normaloff>:/images/06.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButton_7">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/07.png</normaloff>:/images/07.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButton_8">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/05.png</normaloff>:/images/05.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Кисть</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout_2" columnminimumwidth="0,90">
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="spacing">
              <number>2</number>
             </property>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButton_10">
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>90</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>92</width>
                 <height>40</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QPushButton" name="pushButton_9">
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>90</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>92</width>
                 <height>40</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="pushButton_12">
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>90</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>92</width>
                 <height>40</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pushButton_11">
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>90</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>92</width>
                 <height>40</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="pushButton_13">
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>90</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>92</width>
                 <height>40</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="pushButton_14">
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>90</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>92</width>
                 <height>40</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QPushButton" name="pushButton_15">
               <property name="minimumSize">
                <size>
                 <width>90</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>90</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>92</width>
                 <height>40</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Перо</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QGridLayout" name="gridLayout_3" columnstretch="0" columnminimumwidth="0">
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="spacing">
              <number>2</number>
             </property>
             <item row="0" column="0">
              <widget class="QPushButton" name="pushButton_17">
               <property name="maximumSize">
                <size>
                 <width>185</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>185</width>
                 <height>28</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pushButton_18">
               <property name="maximumSize">
                <size>
                 <width>185</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>185</width>
                 <height>28</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="pushButton_19">
               <property name="maximumSize">
                <size>
                 <width>185</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>185</width>
                 <height>28</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QPushButton" name="pushButton_20">
               <property name="maximumSize">
                <size>
                 <width>185</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>185</width>
                 <height>28</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QPushButton" name="pushButton_21">
               <property name="maximumSize">
                <size>
                 <width>185</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>185</width>
                 <height>28</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>753</width>
           <height>629</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
